pps_mw_training/pipelines/cloud_base/input_params.py

```diff
@@ -0,0 +1,309 @@
+        "std": 12.608327215777589,
+    },
+    {
+        "name": "t950",
+        "scale": "linear",
+        "mean": 283.7890126626587,
+        "std": 13.363535983585342,
+    },
+    {
+        "name": "t1000",
+        "scale": "linear",
+        "mean": 286.7742821728768,
+        "std": 14.474404062499847,
+    },
+    {
+        "name": "t_sea",
+        "scale": "linear",
+        "mean": 286.60209551483155,
+        "std": 15.19607293352855,
+    },
+    {
+        "name": "t_land",
+        "scale": "linear",
+        "mean": 286.60209551483155,
+        "std": 15.19607293352855,
+    },
+    {
+        "name": "rh250",
+        "scale": "linear",
+        "mean": 0.005291610806524404,
+        "std": 0.003723000052433534,
+    },
+    {
+        "name": "rh400",
+        "scale": "linear",
+        "mean": 0.006046445369025605,
+        "std": 0.0033096767106421854,
+    },
+    {
+        "name": "rh500",
+        "scale": "linear",
+        "mean": 0.005735704751800622,
+        "std": 0.003239517962875649,
+    },
+    {
+        "name": "rh700",
+        "scale": "linear",
+        "mean": 0.006034013814882565,
+        "std": 0.0029108583874336603,
+    },
+    {
+        "name": "rh850",
+        "scale": "linear",
+        "mean": 0.007134334129411873,
+        "std": 0.0024109007921006044,
+    },
+    {
+        "name": "rh900",
+        "scale": "linear",
+        "mean": 0.007627256122758554,
+        "std": 0.0022071383987135465,
+    },
+    {
+        "name": "rh950",
+        "scale": "linear",
+        "mean": 0.007926371323379571,
+        "std": 0.0021499258670604083,
+    },
+    {
+        "name": "rh1000",
+        "scale": "linear",
+        "mean": 0.00738849284664931,
+        "std": 0.002174385255368654,
+    },
+    {
+        "name": "q250",
+        "scale": "linear",
+        "mean": 6.507605248566506e-05,
+        "std": 7.889838169028413e-05,
+    },
+    {
+        "name": "q400",
+        "scale": "linear",
+        "mean": 0.00046727919108313247,
+        "std": 0.0005378467546237773,
+    },
+    {
+        "name": "q500",
+        "scale": "linear",
+        "mean": 0.001033781902913779,
+        "std": 0.0011316728144658357,
+    },
+    {
+        "name": "q700",
+        "scale": "linear",
+        "mean": 0.0028668664924348104,
+        "std": 0.002547806436435812,
+    },
+    {
+        "name": "q850",
+        "scale": "linear",
+        "mean": 0.005026202192237133,
+        "std": 0.003941367242371887,
+    },
+    {
+        "name": "q900",
+        "scale": "linear",
+        "mean": 0.005886857095257228,
+        "std": 0.004439638445114495,
+    },
+    {
+        "name": "q950",
+        "scale": "linear",
+        "mean": 0.006777255145888057,
+        "std": 0.005117977373985545,
+    },
+    {
+        "name": "q1000",
+        "scale": "linear",
+        "mean": 0.007224722144002037,
+        "std": 0.0054276352054161625,
+    },
+]
+
+
+def get_selected_params(
```

Comment:
check the type hints of this function: float appears twice in the Union

Reply:
Fixed